models/base_model.py

```diff
@@ -14,7 +14,11 @@ class BaseModel(db.Model):
 
     @classmethod
     def find(cls, id):
-        return cls.query.get_or_404(id, f"{cls._name()} not found")
+        object = cls.query.get(id)
+        if object:
+            return object
+        else:
+            raise InvalidRequest(404, f"{cls._name()} not found")
```

Comment:
Just double-checking to make sure I understand this correctly --
We're using a custom `InvalidRequest` Exception here instead of using the flask-sqlalchemy `get_or_404()` method because without flask_restful, the error will be in text rather than json. Is that correct?

Reply:
Correct, but I don't like this implementation.